README.md

```diff
@@ -40,10 +40,12 @@ Retina has two major features:
 Retina can be installed using the Helm chart from GHCR:
 
 ```bash
+# Set the version to a specific version here or get latest version from GitHub API.
+VERSION=$( curl -sL https://api.github.com/repos/microsoft/retina/releases/latest | jq -r .name)
 helm upgrade --install retina oci://ghcr.io/microsoft/retina/charts/retina \
-    --version v0.0.2 \
-		--set image.tag=v0.0.2 \
-		--set operator.tag=v0.0.2 \
+    --version $VERSION \
```

Comment:
is this possible in markdown? i didn't know that 🤔

Reply:
it's not dynamic, it's just for you to copy and paste

Reply:
oops nvm lol